site/content/en/development/design-docs/ipam.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Support node IPAM controller"
```

Comment:
Rename title to "Out-of-tree Node IPAM Controller" (to distinguish from in-tree ipam controller)